felix/bpf/ut/bpf_prog_test.go

```diff
@@ -227,7 +226,20 @@ func setupAndRun(logger testLogger, loglevel, section string, rules *polprog.Rul
 		o(&topts)
 	}
 
-	tempDir, err := ioutil.TempDir("", "calico-bpf-")
+	maps := make([]bpf.Map, len(progMaps))
+	copy(maps, progMaps)
+
+outer:
```

Comment:
Is this change related to the os/io replacement?

Reply:
Ah, these code was few days ago , the code was conflict.
But i see the master branch , have no that code .
i will delete that..